static/app/components/replays/replayController.tsx

```diff
@@ -115,17 +118,23 @@ const ReplayControls = ({
         <Button
           data-test-id="replay-fullscreen"
           size="xsmall"
-          title={t('View in full screen')}
+          title={isFullscreen ? t('Exit full screen') : t('View in full screen')}
+          aria-label={isFullscreen ? t('Exit full screen') : t('View in full screen')}
           icon={<IconResize size="sm" />}
-          onClick={onFullscreen}
-          aria-label={t('View in full screen')}
+          priority={isFullscreen ? 'primary' : undefined}
+          onClick={isFullscreen ? exitFullscreen : onFullscreen}
```

Comment:
Not sure what `onFullscreen` is used for, but do we want to make the callbacks more consistent (e.g. should we have `onExitFullscreen`? or `onFullscreenToggle: (isFullscreen: bool) => void`

Read down further and yeah seems like this is a bit inconsistent?